src/main/scala/com/spotify/spark/bigquery/BigQueryClient.scala

```diff
@@ -109,8 +110,28 @@ private[bigquery] class BigQueryClient(conf: Configuration) {
    */
   def load(gcsPath: String, destinationTable: TableReference,
            writeDisposition: WriteDisposition.Value = null,
-           createDisposition: CreateDisposition.Value = null): Unit = {
+           createDisposition: CreateDisposition.Value = null,
+           isPartitionedByDay: Boolean = false): Unit = {
+
     val tableName = BigQueryStrings.toString(destinationTable)
+    if(isPartitionedByDay) {
+        val datasetId = destinationTable.getDatasetId
+        val projectId: String = destinationTable.getProjectId
+      try {
+        logger.info("Creating Time Partitioned Table")
+        val table = new Table()
+        table.setTableReference(destinationTable)
+        val timePartitioning = new TimePartitioning()
+        timePartitioning.setType("DAY")
+        timePartitioning.setExpirationMs(DEFAULT_TABLE_EXPIRATION_MS)
```

Comment:
Sorry I meant default for time partitioned tables, we can add another parameter for default timing but wont that overcomplicate it? 

Happy with suggestions